examples/webgl_postprocessing_crossfade.html

```diff
@@ -31,155 +29,190 @@
 			import Stats from 'three/addons/libs/stats.module.js';
 			import { GUI } from 'three/addons/libs/lil-gui.module.min.js';
 			import TWEEN from 'three/addons/libs/tween.module.js';
-			import { FullScreenQuad } from 'three/addons/postprocessing/Pass.js';
-
-			let container, stats;
-			let renderer;
-			let transition;
-
-			const transitionParams = {
-				'useTexture': true,
-				'transition': 0,
-				'texture': 5,
-				'cycle': true,
-				'animate': true,
-				'threshold': 0.3
-			};
+			import { EffectComposer } from 'three/addons/postprocessing/EffectComposer.js';
+			import { RenderTransitionPass } from 'three/addons/postprocessing/RenderTransitionPass.js';
+			import { OutputPass } from 'three/addons/postprocessing/OutputPass.js';
+
+			let stats;
+			let renderer, composer, renderTransitionPass;
 
+			const textures = [];
 			const clock = new THREE.Clock();
 
+			const params = {
+				sceneAnimate: true,
+				transitionAnimate: true,
+				transition: 0,
+				useTexture: true,
+				texture: 5,
+				cycle: true,
+				threshold: 0.1,
+			};
+
+			const fxSceneA = new FXScene( new THREE.BoxGeometry( 2, 2, 2 ), new THREE.Vector3( 0, - 0.4, 0 ), 0xffffff );
+			const fxSceneB = new FXScene( new THREE.IcosahedronGeometry( 1, 1 ), new THREE.Vector3( 0, 0.2, 0.1 ), 0x000000 );
+
 			init();
 			animate();
 
 			function init() {
 
 				initGUI();
+				initTextures();
 
-				container = document.getElementById( 'container' );
+				const width = window.innerWidth;
+				const height = window.innerHeight;
+				const dpr = window.devicePixelRatio;
 
 				renderer = new THREE.WebGLRenderer( { antialias: true } );
-				renderer.setPixelRatio( window.devicePixelRatio );
-				renderer.setSize( window.innerWidth, window.innerHeight );
-				container.appendChild( renderer.domElement );
+				renderer.setPixelRatio( dpr );
+				renderer.setSize( width, height );
+				renderer.setAnimationLoop( animate );
+				document.body.appendChild( renderer.domElement );
+
+				composer = new EffectComposer( renderer );
 
 				stats = new Stats();
-				container.appendChild( stats.dom );
+				document.body.appendChild( stats.dom );
 
-				const geometryA = new THREE.BoxGeometry( 2, 2, 2 );
-				const geometryB = new THREE.IcosahedronGeometry( 1, 1 );
-				const sceneA = new FXScene( geometryA, new THREE.Vector3( 0, - 0.4, 0 ), 0xffffff );
-				const sceneB = new FXScene( geometryB, new THREE.Vector3( 0, 0.2, 0.1 ), 0x000000 );
+				renderTransitionPass = new RenderTransitionPass( fxSceneA.scene, fxSceneA.camera, fxSceneB.scene, fxSceneB.camera );
+				renderTransitionPass.setPixelRatio( dpr );
+				renderTransitionPass.setSize( width, height );
```

Comment:
```js
renderTransitionPass.setPixelRatio( dpr );
renderTransitionPass.setSize( width, height );
```
It should not be necessary to have these two lines. Let the composer resize the pass to the correct size.

Reply:
Done.